console/frontend/src/main/frontend/src/app/views/logging/logging.component.ts

```diff
@@ -84,20 +87,28 @@ export class LoggingComponent implements OnInit {
   };
 
   openDirectory(directory: string) {
-    this.loggingService.getLogging(directory).subscribe({ next: (data) => {
-      this.alert = false;
-      Object.assign(this, data);
-      this.path = data.directory;
-      if (data.count > data.list.length) {
-        this.alert = "Total number of items [" + data.count + "] exceeded maximum number, only showing first [" + (data.list.length - 1) + "] items!";
+    this.loggingService.getLogging(directory).subscribe({
+      next: (data) => {
+        this.alert = false;
+        this.originalList = data.list;
+        this.sortedlist = data.list;
+        this.directory = data.directory;
+        this.path = data.directory;
+        if (data.count > data.list.length) {
+          this.alert = "Total number of items [" + data.count + "] exceeded maximum number, only showing first [" + (data.list.length - 1) + "] items!";
+        }
+      }, error: (data) => {
+        this.alert = (data.error) ? data.error.error : "An unknown error occured!";
```

Comment:
```suggestion
        this.alert = data.error?.error || "An unknown error occured!";
```
Maybe a bit nicer syntactically